drs-auth_client/lib/drs/auth_client/client.rb

```diff
@@ -0,0 +1,81 @@
+require "active_support/core_ext/hash/keys"
+require "active_support/core_ext/string/inflections"
+require "faraday"
+
+require "drs/auth_client/models/organisation"
+require "drs/auth_client/models/profile"
+
+module Drs
+  module AuthClient
+    class Client
+      def initialize(auth_token)
+        @auth_token = auth_token
+      end
+
+      def get(path)
+        response = conn.get do |request|
+          request.url path
+          request.headers['Authorization'] = "Bearer #{@auth_token}"
+        end
+
+        if response.status == 200
+          response.body
+        else
+          nil
+        end
```

Comment:
What happens when the response is not a 200... Is there a faraday exception raised (or some other type?) or is just nil returned in which case we will end up calling nil.has_key? when parsing the response in the BaseModel?